Makefile

```diff
@@ -30,8 +30,8 @@ all: build unit-test system-test ubuntu-tests
 # 'all-CI' target is used by the scripts/CI.sh that passes appropriate set of
 # ENV variables (from the jenkins job) to run OS (centos, ubuntu etc) and
 # sandbox specific(vagrant, docker-in-docker)
-all-CI: stop clean start
-	make ssh-build
+all-CI: stop clean
+	make checks run-build start ssh-build
```

Comment:
This means that the host needs to have build tools installed AND that every time we upgrade the build tools (e.g. go ver) we will need to update every single host.